backend/db/init_db.py

```diff
@@ -7,18 +7,18 @@ def _init_postgres_db(app):
     db.init_app(app)
     migrate.init_app(app, db)
 
-    # Clear database tables
-    db.reflect()
-    db.drop_all()
+    # # Clear database tables
+    # db.reflect()
+    # db.drop_all()
 
-    # Init database tables
-    db.create_all()
+    # # Init database tables
+    # db.create_all()
 
-    db.session.commit()
+    # db.session.commit()
 
-    # TODO: remove when we have real data
-    # Seed database with sample data
-    seed()
+    # # TODO: remove when we have real data
+    # # Seed database with sample data
+    # seed()
```

Comment:
Is this so that the `seed` method is commented out by default?

Reply:
I think there was some mention somewhere about adding a flag when running the server to specify whether to seed the database. Perhaps we can go with that?

Reply:
Yeah, I think we mentioned that in the future, we wanted to have it seed the DB depending on the environment that we deploy to.